### PR TITLE
More secure captcha

### PR DESCRIPTION
**Pull request by Crabcyborg**

### CODE REVIEW

File: classes/views/frm-entries/form.php [hunk 1]
```diff
@@ -34,6 +34,8 @@
 <input type="hidden" name="frm_hide_fields_<?php echo esc_attr( $form->id ); ?>" id="frm_hide_fields_<?php echo esc_attr( $form->id ); ?>" value="<?php echo esc_attr( $frm_hide_fields ); ?>" />
 <input type="hidden" name="form_key" value="<?php echo esc_attr( $form->form_key ); ?>" />
 <input type="hidden" name="item_meta[0]" value="" />
+<?php FrmFieldCaptcha::render_checked_response(); ?>
```
Comment (stephywells) [inline classes/views/frm-entries/form.php:37]: Can we call this using the frm_after_title hook or frm_entry_form to avoid the extra code in lite?

Comment (Crabcyborg) [reply classes/views/frm-entries/form.php]: Awesome, changed that. Love all of these actions and filters, just getting the hang of using them properly still :)

Comment (stephywells) [reply classes/views/frm-entries/form.php]: That sounds like a good idea to me.

Comment (stephywells) [reply classes/views/frm-entries/form.php]: well, actually, maybe it's not worth it. We'll need to check it specifically anyway. I think it's fine as is.

**Pull request merged**